components/ui/menu/SidebarMenu.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <nav class="sidebar-menu">
+    <h2 class="sidebar-menu__title">
+      Elements
+    </h2>
+    <MenuLink
+      v-for="link in qiskitElements"
+      :key="link.url"
+      :class="['sidebar-menu__link', isActive(link) && 'sidebar-menu__link_active']"
+      v-bind="link"
+    />
+    <h2 class="sidebar-menu__title">
+      Learn more
+    </h2>
+    <MenuLink
+      class="sidebar-menu__link"
+      v-bind="communityLink"
+    />
+    <div
+      v-if="isCommunityActive()"
+      class="sidebar-menu__second-level"
+    >
+      <MenuLink
+        v-for="link in communitySubLinks"
+        :key="link.url"
+        :class="['sidebar-menu__link', 'sidebar-menu__link_secondary', isActive(link) && 'sidebar-menu__link_active']"
+        v-bind="link"
+      />
+    </div>
+    <MenuLink
+      v-for="link in learnMore"
+      :key="link.url"
+      :class="['sidebar-menu__link', isActive(link) && 'sidebar-menu__link_active']"
+      v-bind="link"
+    />
+  </nav>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import MenuLink from '~/components/ui/menu/MenuLink.vue'
+import menuMixin from '~/mixins/menu'
+
+@Component({
+  components: { MenuLink },
+  mixins: [menuMixin]
```

Comment:
Non-blocking: we should try to use the [`Mixins` notation](https://github.com/kaorun343/vue-property-decorator#usage).